python/test/unit/language/test_core.py

```diff
@@ -3146,35 +3154,43 @@ def convert_fp8_to_fp32(x, device, dtype_str):
 
 @pytest.mark.interpreter
 @pytest.mark.parametrize(
-    "M, N, K, num_warps, col_a, col_b, epilogue, input_precision, in_dtype, out_dtype, kpack",
-    [(*shape, 4, False, False, epilogue, input_precision, in_dtype, out_dtype, 1)
+    "M, N, K, num_warps, col_a, col_b, epilogue, input_precision, in_dtype, out_dtype, kpack, mma_nonk_size",
+    [(*shape, 4, False, False, epilogue, input_precision, in_dtype, out_dtype, 1, None)
```

Comment:
This whole list is really difficult to manage now. Can you send out a pull request to refactor it a bit into defining some variables to hold separate lists as a prelimiary step? And then we can rebase this pull request on top of that to easily add/change.

Reply:
> This whole list is really difficult to manage now.

That is fair.

> Can you send out a pull request to refactor it a bit into defining some variables to hold separate lists as a prelimiary step

I think the other way is better: do this PR first, then refactor test list or include everything in this PR.
This approach has two benefits:
- faster delivery of a bugfix, which I believe have priority over refactoring
- less work for me rebasing changes and less opportunities to break something along the way

Reply:
I can split this commit in a separate PR: https://github.com/triton-lang/triton/pull/5406/commits/85bb174e53a779ab9ca974e89d105acd915605c8